build.gradle.kts

```diff
@@ -39,6 +39,8 @@ dependencies {
     compileOnly("org.kohsuke:access-modifier-checker:1.21")
     annotationProcessor(sezpoz)
     implementation(gradleApi())
+    compileOnly("org.eclipse.jgit:org.eclipse.jgit:5.13.1.202206130422-r")
+    testImplementation("org.eclipse.jgit:org.eclipse.jgit:5.13.1.202206130422-r")
```

Comment:
Maybe extract dependency into a variable to be able to reuse it? Similar to the `sezpoz` dependency.

Reply:
done